lms/static/sass/shared-v2/_header.scss

```diff
@@ -0,0 +1,80 @@
+// edX theme: Global Header
+// ====================
+
+.header-global {
+  @extend %ui-depth1;
+  @include box-sizing(border-box);
+  position: relative;
+  width: 100%;
+  border-bottom: 4px solid $courseware-border-bottom-color;
+  box-shadow: 0 1px 5px 0 $shadow-l1;
```

Comment:
Because it's not really reused I'm not sure of the value in making it a variable. There is the UI depth section in the PL but I don't think this PR should be updating the header styles (due to it having to match non-PL pages for the foreseeable future.